### PR TITLE
Precompile Handlebars templates and partials

### PR DESCRIPTION
**Pull request by mohd-akram**
This improves performance significantly (got up to 20x).

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -368,7 +379,20 @@ function fastifyView (fastify, opts, next) {
     })
   }
 
-  next()
+  if (prod && type === 'handlebars' && options.partials) {
```
Comment (Eomm) [inline index.js]: why this changes in the `mustache` handler?

Comment (mohd-akram) [reply index.js]: The only change is just to update the function name there.

**Comment by mohd-akram** [comment]
Could this be merged?

**Comment by mcollina** [comment]
@Eomm?

**Review by lilpoozie2005 — APPROVED** [review]
